java/offline-geocode/src/main/java/com/esri/arcgisruntime/sample/offlinegeocode/MainActivity.java

```diff
@@ -224,7 +224,6 @@ public void run() {
     mPinSourceSymbol.setWidth(20);
     mPinSourceSymbol.loadAsync();
     mPinSourceSymbol.setLeaderOffsetY(45);
-    mPinSourceSymbol.setOffsetY(-48);
```

Comment:
I think what @hardikc07 is looking for here is .setOffsetY(-45). Keeping the callout at the top of the pin allows the user to drag the pin around to reverse geocode.

Reply:
I spoke to Hardik and we see 2 problems here:
1. the red pin location and touch are not in sync. the red pin is always some points lower on Y axis than the touch. This is set by the offsetY on mPinSourceSymbol.
2. I agree to your suggestion that the callout should be placed bit higher than the red pin. 
This is set by the onLongPress function, in the variable screenpoint. Line 459. 

To address the above two points, I set the mPinSourceSymbol.setOffsetY to -25 and screenPoint Y coordinate to -75 [e.getY()-75]. Is there a better way to do this? since it can vary for different phone screen sizes?

This way i am able to see the red pin on the touch point and also the callout on top of the red pin.